lib/core/versions/latest/AnchorFile.ts

```diff
@@ -167,9 +167,9 @@ export default class AnchorFile {
 
     const createOperations = createOperationArray.map(operation => {
       return {
-        suffix_data: { 
-          delta_hash: operation.suffixData.deltaHash, 
-          recovery_commitment: operation.suffixData.recoveryCommitment, 
+        suffixData: { 
+          deltaHash: operation.suffixData.deltaHash, 
+          recoveryCommitment: operation.suffixData.recoveryCommitment, 
           type: operation.suffixData.type 
```

Comment:
Now that we don't need this transform logic, let's just assign the property directly and remove the bloat?
Do the same thing for all occurrences to remove all the bloat.
```suggestion
        suffixData: operation.suffixData
```